src/FunctionsClient.ts

```diff
@@ -11,6 +11,7 @@ import {
 export class FunctionsClient {
   protected url: string
   protected headers: Record<string, string>
+  protected region: string
```

Comment:
hmm..what's the API to set the `region`?

Reply:
I am not sure I follow.